src/Nancy/Helpers/ProxyNancyReferenceProber.cs

```diff
@@ -0,0 +1,18 @@
+#if !CORE
+namespace Nancy.Helpers
+{
+    using System;
+    using System.Reflection;
+    using Nancy.Extensions;
+
+    internal class ProxyNancyReferenceProber : MarshalByRefObject
+    {
+        public bool HasReference(AssemblyName assemblyNameForProbing, AssemblyName referenceAssemblyName)
+        {
+            var assemblyForInspection = Assembly.Load(assemblyNameForProbing);

```

Comment:
Ok, then yes. I tested it and it came out correctly, I just pushed these changes.